tests/testthat/test-csv.R

```diff
@@ -373,14 +373,14 @@ test_that("read_cmdstan_csv() reads adaptation step size correctly", {
   csv_files <- test_path("resources", "csv", "model1-2-no-warmup.csv")
 
   csv_out <- read_cmdstan_csv(csv_files)
-  expect_equal(csv_out$step_size[[2]], 0.672434)
+  expect_equal(csv_out$step_size[[as.character(2)]], 0.672434)
 
   csv_files <- c(test_path("resources", "csv", "model1-1-dense_e_metric.csv"),
                  test_path("resources", "csv", "model1-2-dense_e_metric.csv"))
 
   csv_out <- read_cmdstan_csv(csv_files)
-  expect_equal(csv_out$step_size[[1]], 0.11757)
-  expect_equal(csv_out$step_size[[2]], 0.232778)
+  expect_equal(csv_out$step_size[[as.character(1)]], 0.11757)
+  expect_equal(csv_out$step_size[[as.character(2)]], 0.232778)
```

Comment:
These two don't actually need to be changed since they're still the 1st and 2nd element of the list, but probably good to be consistent so let's keep the changed version.